definitions/firefox_ios.toml

```diff
@@ -31,6 +31,26 @@ description = "The number of days in an observation window that clients used the
 select_expression = "COUNT(DISTINCT DATE(submission_timestamp))"
 data_source = "baseline"
 
+
+[metrics.daily_active_users]
+data_source = "baseline"
```

Comment:
Similar comment to above, I believe we would be missing any baseline data from ios beta channel here AFAICT that data lives in the ` moz-fx-data-shared-prod.org_mozilla_ios_firefoxbeta` associated tables. The all-channel unioned data would be in `moz-fx-data-shared-prod.firefox_ios` (again, IIUC).